eZ/Publish/Core/Repository/Mapper/ContentMapper.php

```diff
@@ -159,6 +167,42 @@ public function mapFieldsForUpdate(
         return $fields;
     }
 
+    public function getFieldValueForCreate(FieldDefinition $fieldDefinition, ?Field $field): Value
+    {
+        if (isset($field)) {
```

Comment:
This is not strict, did you mean:
```suggestion
        if (null !== $field) {
```
?